node-tests/blueprints/route-test.js

```diff
@@ -395,7 +391,6 @@ describe('Acceptance: ember generate and destroy route', function() {
     var args = ['route', 'application', '--pod'];
 
     return emberNew()
-      .then(() => remove(path.resolve('app/templates/application.hbs')))
```

Comment:
@kellyselden - Do we just not need to remove this file?

Reply:
Yea. I tried converting to `fs.unlink` first, but the file is never there. Must be legacy, and can come back in the future if needed.